internal/db/2pc.go

```diff
@@ -4,12 +4,23 @@ import (
 	"context"
 	"fmt"
 
-	"github.com/google/uuid"
 	"github.com/jmoiron/sqlx"
 )
 
-func PrepareTransaction(ctx context.Context, dbConn sqlx.ExecerContext, txID uuid.UUID) error {
+func PrepareTransaction(ctx context.Context, dbConn sqlx.ExecerContext, txID fmt.Stringer) error {
```

Comment:
Perhaps for greater uniformity, it is worth changing the stringer to a string.

Reply:
Signature updated